assets/src/models/ladderVehicle.ts

```diff
@@ -1,40 +1,23 @@
 import { TimepointStatusYFunc } from "../components/ladder"
-import { partition } from "../helpers/array"
-import vehicleLabel from "../helpers/vehicleLabel"
-import featureIsEnabled from "../laboratoryFeatures"
-import { Ghost, Vehicle, VehicleId, VehicleOrGhost } from "../realtime"
-import { ViaVariant } from "../schedule"
-import { Settings } from "../settings"
+import { Ghost, Vehicle, VehicleOrGhost } from "../realtime"
 import {
   directionOnLadder,
   LadderDirection,
   VehicleDirection,
 } from "./ladderDirection"
-import { hasBlockWaivers, isVehicle } from "./vehicle"
-import { DrawnStatus, drawnStatus, HeadwaySpacing } from "./vehicleStatus"
+import { isVehicle } from "./vehicle"
 
 export interface LadderVehicle {
-  vehicleId: VehicleId
-  label: string
-  viaVariant: ViaVariant | null
-  status: DrawnStatus
-  hasBlockWaivers: boolean
-  headwaySpacing: HeadwaySpacing | null
+  vehicle: VehicleOrGhost
   x: number
   y: number
   vehicleDirection: VehicleDirection
   scheduledY?: number
   scheduledVehicleDirection?: VehicleDirection
-  lane: number
```

Comment:
Why does this go away?

Reply:
Yeah, I wasn't quite sure what to do about this. It's never used outside of this file, so I figured it shouldn't be part of the public interface, but the way the LadderVehicle object is created with ` return {...vehicleInLane, x}`, technically the lane still in the object when it's returned.

Reply:
Yeah, personally I'd just describe the data completely and let other things worry about what they do or do not want to use, but I don't feel strongly.

Reply:
I looked at adding it back, but it requires adding unused fields to a bunch of test data, too. That's not a big deal, but I don't think it's worth it to make the interface reflect the implementation when ideally it should probably be the implementation that changes to reflect the interface we'd use.